doc/how_to_guides/isaac_panda/isaac_panda_tutorial.rst

```diff
@@ -0,0 +1,169 @@
+      <xacro:if value="${use_fake_hardware}">
+        <plugin>mock_components/GenericSystem</plugin>
+      </xacro:if>
+      <xacro:unless value="${use_fake_hardware}">
+        <plugin>franka_hardware/FrankaHardwareInterface</plugin>
+        <param name="robot_ip">${robot_ip}</param>
+      </xacro:unless>
+    </hardware>
+
+
+`Hardware Components <https://control.ros.org/master/doc/getting_started/getting_started.html#hardware-components>`_
+can be of different types, but the plugin ``<plugin>mock_components/GenericSystem</plugin>`` is very a simple ``System``
+that forwards the incoming ``command_interface`` values to the tracked ``state_interface`` of the joints (I.E. perfect control of the simulated joints).
+
+For us to expand our Panda robot to Isaac Sim we first have to introduce `isaac_ros2_control <https://github.com/PickNikRobotics/isaac_ros2_control>`_.
+This Hardware Interface is a ``System`` that subscribes and publishes on a configured topics.
+For this tutorial the topic ``/isaac_joint_states`` will be the robots current state and ``/isaac_joint_commands`` will be used to actuate it.
+The `moveit_resources_panda_moveit_config <https://github.com/ros-planning/moveit_resources/blob/humble/panda_moveit_config/config/panda.ros2_control.xacro#L7>`_
+we are using in this tutorial does not support connecting to hardware, so our ``ros2_control.xacro`` is now
+updated to load the ``IsaacSystem`` plugin when the flag ``ros2_control_hardware_type`` is set to ``isaac``.
+
+.. code-block:: XML
+
+    <xacro:if value="${ros2_control_hardware_type == 'mock_components'}">
+        <plugin>mock_components/GenericSystem</plugin>
+    </xacro:if>
+    <xacro:if value="${ros2_control_hardware_type == 'isaac'}">
+        <plugin>isaac_ros2_control/IsaacSystem</plugin>
+        <param name="joint_commands_topic">/isaac_joint_commands</param>
+        <param name="joint_states_topic">/isaac_joint_states</param>
+    </xacro:if>
+
+In this tutorial we have included an Isaac python script that loads a Panda robot
+and builds an `OnmiGraph <https://docs.omniverse.nvidia.com/prod_extensions/prod_extensions/ext_omnigraph.html>`_
+to publish and subscribe to the ros topics used to control the robot.
+To learn about configuring your Isaac robot to communicate with ROS2 please see the
+`Joint Control tutorial <https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_manipulation.html>`_
+on Omniverse.
+
+Computer Setup
+--------------
+
+1. Install `Isaac Sim <https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/install_workstation.html>`_.
+
+2. For RViz to run inside Docker we have to tell the host to allow it to display applications.
+
+.. note:: this step is only required once every host machine login
+
+.. code-block:: bash
+
+  xhost +local:docker
+
+1. Perform a shallow clone of the MoveIt2 Tutorials repo.
+
+.. code-block:: bash
+
+  git clone https://github.com/ros-planning/moveit2_tutorials.git -b humble --depth 1
```

Comment:
You need to use my branch until this is merged.
`git clone https://github.com/MarqRazz/moveit2_tutorials.git -b pr-isaac_tutorial --depth 1`